Mithril update 2023-12-06 (#300)
input-output-hk/cardano-updates: blog/2023-12-06-mithril.md

---
title: Mithril Team Update
slug: 2023-12-06-mithril
authors: jpraynaud
tags: [mithril]
hide_table_of_contents: false
---

## High level overview

> This week, the Mithril team made progress on adapting the Mithril client library for WASM compilation and providing a Javascript API for it. They also completed the optimization of the performance of the aggregator. Additionally, the team started to implement a testing Mithril network for SanchoNet, as well as conducting threat modeling and risk analysis for P2P networking.

Reviewer: ```suggestion
This week, the Mithril team made progress in adapting the Mithril client library for WASM compilation and providing a JavaScript API for it. They also completed the optimization of the aggregator's performance. Additionally, the team initiated the implementation of a testing Mithril network for SanchoNet and conducted threat modeling and risk analysis for P2P networking.
```
